Header header

float32 tgtAngle
float32 tgtMagnitude
int32 tgtLandmark
float32 tgtCamDistance
float32 tgtIrDistance

float32 curAngle
bool ignoreAngle

int32 curLandmark
float32 curCamDistance

float32 motorR
float32 motorL

float32 IRLDist
float32 IRRDist

float32 camAngle
bool camCentered
